cme/protocols/smb/atexec.py

```diff
@@ -203,7 +203,7 @@ def execute_handler(self, command, fileless=False):
                         break
                     except Exception as e:
                         if tries >= self.__tries:
-                            self.logger.fail(f'ATEXEC: Get output file error, maybe got detected by AV software, please increase the number of tries with the option "--get-output-tries". If it\'s still failing maybe something is blocking the schedule job, try another exec method')
+                            self.logger.fail(f'ATEXEC: Couldn\'t retrieve output file, maybe got detected by AV. Please increase the number of tries with the option "--get-output-tries". If it\'s still failing, try the wmi protocol or another exec method')
```

Comment:
This is why I like double quotes on the outside, so you can use single quotes inside without escaping. I think we were trying to stick to doubles on the outside, no?

Reply:
There's a few instances of this too ^^

Reply:
Agreed, I will change that